api/modules/files/routes/read.js

```diff
@@ -28,19 +30,19 @@ module.exports = [{
     }
   }
 }, {
-  method: 'GET',
-  path: '/projects/{project_id}/files',
+  method: `GET`,
+  path: `/projects/{project_id}/files`,
   config: {
     pre: [
-      prereqs.confirmRecordExists(Model, {
-        mode: 'param',
-        requestKey: 'project_id'
+      Prerequisites.confirmRecordExists(FilesModel, {
+        mode: `param`,
+        requestKey: `project_id`
       }),
-      prereqs.validateUser(),
-      prereqs.validateOwnership()
+      Prerequisites.validateUser(),
+      Prerequisites.validateOwnership()
     ],
-    handler: controller.getAll.bind(controller),
-    description: 'Retrieve a collection of file objects that belong to a single project object, based on `project_id`.',
+    handler: filesController.getAll.bind(filesController),
```

Comment:
`handler: args => filesController.getAll(...args),` although it feels odd that this can't just be a static function and needs explicit tying to a specific fileController instance. It suggests we might have unnecessary data capturing in the fileController class.